src/csharp/Microsoft.Spark.E2ETest/Microsoft.Spark.E2ETest.csproj

```diff
@@ -5,13 +5,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="coverlet.msbuild" Version="2.4.0">
+    <PackageReference Include="coverlet.msbuild" Version="2.6.1">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
```

Comment:
Why can we remove these now?

Reply:
The need to be because they are automatically included by arcade when the project is a "test" project. If I don't remove them I get errors saying it was referenced twice. (once implicitly)

Reply:
Ah I see, that is why I couldn’t find where these references were defined. (Arcade magic)